portfolio/src/main/webapp/index.html

```diff
@@ -25,7 +25,7 @@
       <div id="homecontent">
         <div class="scrollmenu">
           <button class="scrollbutton" onclick="getCommentsTag('#general');">#general</button>
-          <button class="scrollbutton" onclick="getCommentsTag('#animals');">#animals</button>
+          <button class="scrollbutton" onclick="getCommentsTag('#animal');">#animal</button>
```

Comment:
Please remember to update it in InternalTags also

Reply:
that's how it is in internal tags(: